yt_dlp/postprocessor/embedthumbnail.py

```diff
@@ -79,14 +79,15 @@ def run(self, info):
 
         original_thumbnail = thumbnail_filename = info['thumbnails'][idx]['filepath']
 
-        # Convert unsupported thumbnail formats to PNG (see #25687, #25717)
-        # Original behavior was to convert to JPG, but since JPG is a lossy
-        # format, there will be some additional data loss.
-        # PNG, on the other hand, is lossless.
         thumbnail_ext = os.path.splitext(thumbnail_filename)[1][1:]
-        if thumbnail_ext not in ('jpg', 'jpeg', 'png'):
-            thumbnail_filename = convertor.convert_thumbnail(thumbnail_filename, 'png')
-            thumbnail_ext = 'png'
+        if info['ext'] not in ['mkv', 'mka']:
+            # Convert unsupported thumbnail formats to PNG (see #25687, #25717)
+            # Original behavior was to convert to JPG, but since JPG is a lossy
+            # format, there will be some additional data loss.
+            # PNG, on the other hand, is lossless.
+            if thumbnail_ext not in ('jpg', 'jpeg', 'png'):
+                thumbnail_filename = convertor.convert_thumbnail(thumbnail_filename, 'png')
+                thumbnail_ext = 'png'
```

Comment:
```suggestion
        # Convert unsupported thumbnail formats to PNG (see #25687, #25717)
        # Original behavior was to convert to JPG, but since JPG is a lossy
        # format, there will be some additional data loss.
        # PNG, on the other hand, is lossless.
        if info['ext'] not in ('mkv', 'mka') and thumbnail_ext not in ('jpg', 'jpeg', 'png'):
            thumbnail_filename = convertor.convert_thumbnail(thumbnail_filename, 'png')
            thumbnail_ext = 'png'
```